engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/rollingformula/BaseRollingFormulaOperator.java

```diff
@@ -172,8 +172,8 @@ protected static IntConsumer getChunkSetter(
                     "Output chunk type should not be Boolean but should have been reinterpreted to byte");
         }
         if (chunkType == ChunkType.Byte) {
+            final WritableByteChunk<? extends Values> writableChunk = valueChunk.asWritableByteChunk();
```

Comment:
Maybe you can do this for other types as well in the following lines.
Then you can also change it to a lambda
```
        if (chunkType == ChunkType.Byte) {
            final WritableByteChunk<? extends Values> writableChunk = valueChunk.asWritableByteChunk();
            return value -> writableChunk.set(value, formulaOutputSource.getByte(0));
        }
```
Also, in the spirit of beautifying code, we can also change the name from `i`.

Reply:
Also look at https://github.com/deephaven/deephaven-core/pull/6143#discussion_r1787942012